### ISSUE TITLE
://flags

### ISSUE THREAD

**Issue by davidraywilson** [wexond/browser-base#158]
Is it possible to use chromes ://flags feature in electron? Specifically, as a Windows user I really like the 'Overlay Scrollbars' feature so the scrollbars will auto hide.

**Comment by sentialx** [comment]
Unfortunately not, but some of the flags will be implemented in the future.

**Comment by danimesq** [comment]
@sentialx
> Unfortunately not, but some of the flags will be implemented in the future.

Yes, its possible to enable Chromium flags from your JS file in Electron.
